iphone/Classes/TiUIWebView.m

```diff
@@ -7,66 +7,19 @@
 #ifdef USE_TI_UIWEBVIEW
```

Comment:
Not explicitly reviewing this file, since it should be a 1:1 copy of the existing file from Ti.WKWebView + the macros. If not, please change.